sympy/assumptions/cnf.py

```diff
@@ -0,0 +1,373 @@
+from itertools import combinations, product
```

Comment:
Add a docstring to the top of the file explaining that the things in this file are for internal use of the assumptions system only. For everything else, people should use the classes in sympy.logic.